specification/synapse/resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/sqlServer.json

```diff
@@ -0,0 +1,1702 @@
+            "$ref": "./examples/CreateWorkspaceManagedSqlServerExetendedBlobAuditingSettingsWithMinParameters.json"
+          },
+          "Create or update workspace managed sql server's extended blob auditing policy of with all parameters": {
+            "$ref": "./examples/CreateWorkspaceManagedSqlServerExtendedBlobAuditingSettingsWithAllParameters.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Synapse/workspaces/{workspaceName}/extendedAuditingSettings": {
+      "get": {
+        "operationId": "WorkspaceManagedSqlServerExtendedBlobAuditingPolicies_ListByWorkspace",
+        "summary": "List server's extended blob auditing policies.",
+        "description": "List workspace managed sql server's extended blob auditing policies.",
+        "tags": [
+          "WorkspaceManagedSqlServer, BlobAuditing"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./workspace.json#/parameters/workspaceNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved extended auditing settings.",
+            "schema": {
+              "$ref": "#/definitions/ExtendedServerBlobAuditingPolicyListResult"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 404 SubscriptionDoesNotHaveServer - The requested server was not found\n\n * 404 ServerNotInSubscriptionResourceGroup - Specified server does not exist in the specified resource group and subscription.\n\n * 409 UnsupportedWorkspaceType - Unsupported operation for the workspace type.n\n * 500 InternalServerError - Something went wrong."
+          }
+        },
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "Get workspace managed sql server's extended blob auditing settings": {
+            "$ref": "./examples/ListWorkspaceManagedSqlServerExtendedBlobAuditingSettings.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Synapse/workspaces/{workspaceName}/securityAlertPolicies/{securityAlertPolicyName}": {
+      "get": {
+        "operationId": "WorkspaceManagedSqlServerSecurityAlertPolicy_Get",
+        "summary": "Get server's security alert policy.",
```

Comment:
Get a workspace managed sql server's security alert policy

Reply:
then swagger will throw an error stating summary and description is same